core/queries/GetFavouriteArticles.js

```diff
@@ -0,0 +1,16 @@
+import gql from "graphql-tag";
+
+export default gql`
+  {
+    verifiedUser {
```

Comment:
What does `verifiedUser` mean? Is the current user? Can we use something more meaningful like `currentUser` or shorter like `me`.